lib/segment/src/common/rocksdb_wrapper.rs

```diff
@@ -220,15 +220,9 @@ impl DatabaseColumnWrapper {
         Box::new(move || {
             let db = database.read();
             let Some(column_family) = db.cf_handle(&column_name) else {
-                // It is possible, that the index was removed during the flush by user or another thread.
-                // In this case, non-existing column family is not an error, but an expected behavior.
-
-                // Still we want to log this event, for potential debugging.
-                log::warn!(
-                    "Flush: RocksDB cf_handle error: Cannot find column family {}. Ignoring",
-                    &column_name
-                );
-                return Ok(()); // ignore error
```

Comment:
We don't want to ignore it here for 2 reasons:

- `rocksdb_wrapper` is used in other structures as well, not only in payload index. Some of them can't be removed, so ignoring error is not correct.
- There are other functions in payload index flush (e.g. `remove` in `DatabaseColumnScheduledDeleteWrapper`) which are trying to read column family. They can fail in the same way, but previously we didn't ignore those other errors. 


Instead, we not handle this type error in the segment level flush for the whole `payload_index` structure, and only for `payload_index`